lib/node_modules/@stdlib/blas/ext/base/ndarray/gsumors/docs/repl.txt

```diff
@@ -0,0 +1,32 @@
+
+{{alias}}( arrays )
+    Computes the sum of all elements in a one-dimensional ndarray
+    using ordinary recursive summation.
```

Comment:
```suggestion
    Computes the sum of all elements in a one-dimensional ndarray using ordinary
    recursive summation.
```

@MeKaustubh07 It would be very much appreciated if you could self-verify that REPL text files are wrapped appropriately. This should be straightforward to do visually in your local IDE. E.g., if you use VS Code, you can configure rulers to display in your editor view. Otherwise, it is rather tedious needing to manually check whether each REPL text file you add is appropriately wrapped.